### ISSUE TITLE
Bundle supports which version of Laravel

### ISSUE THREAD

**Issue by ghost** [UserScape/Laravel-Bundles#44]
Could do with some way of indicating which version of Laravel the bundle supports. 

Could prompt the contributor when they initially uploaded it and going forward give the ability to add versions in a bundle management area.

**Comment by taylorotwell** [comment]
Yeah I think this is a good idea. Also I plan to support versioned bundles in next Laravel release.
